.changelog/config.toml

```diff
@@ -14,4 +14,4 @@ release_date_formats = [
 ]
 
 [change_set_sections]
-sort_entries_by = "entry-text"
+sort_entries_by = "id"
```

Comment:
I'm curious why this was changed.

Reply:
I explained above, I think this change was added and the changelog was not re-generated after. It messes up with the current ordering, in the past I believe it was the id which makes more sense from a chronological perspective.